src/schedule-and-details/schedule-section/certificate-display-row/CertificateDisplayRow.scss

```diff
@@ -0,0 +1,39 @@
+.dropdown-certificate {
+  .dropdown-certificate-label {
+    font: normal $font-weight-bold .75rem/1.25rem $font-family-base;
+    color: $gray-500;
+    margin-bottom: .5rem;
+  }
+
+  .dropdown-certificate-control {
+    .dropdown-certificate-control-toggle {
+      font-size: $input-font-size;
+      font-weight: $input-font-weight;
+      line-height: $input-line-height;
+      background: $input-bg;
+      border-color: $input-border-color;
+      border-width: $input-border-width;
+      box-shadow: $input-box-shadow;
+      border-radius: $input-border-radius;
+      color: $input-color;
+      padding: $input-padding-y $input-padding-x;
+      height: $input-height;
+      justify-content: space-between;
+      width: 100%;
+
+      &:not(:disabled):not(.disabled):active,
+      &:not(:disabled):not(.disabled).active,
+      .show & {
+        color: $input-color;
+        border-color: $input-bg;
+        background-color: $input-bg;
+      }
+    }
+  }
+
+  .pgn__form-control-description {
+    font: normal $font-weight-normal .75rem/1.25rem $font-family-base;
```

Comment:
Is it possible to use [Paragon](https://paragon-openedx.netlify.app/foundations/typography) fonts here?

Reply:
lets leave as it is i've already moved it to another common styles file